README.md

```diff
@@ -18,9 +18,9 @@
 
   * **Isabelle**
 
-      - Download [Isabelle2023](https://isabelle.in.tum.de/website-Isabelle2023/dist/Isabelle2023_linux.tar.gz)
+      - Download [Isabelle2024](https://isabelle.in.tum.de)
```

Comment:
Please use https://isabelle.in.tum.de/website-Isabelle2024/dist/Isabelle2024_linux.tar.gz